README.md

```diff
@@ -18,12 +18,14 @@ cd iOS
 
 ## install cocoapods via homebrew, use that
 brew install cocoapods
+/opt/homebrew/opt/ruby/bin/gem install cocoapods-acknowledgements
```

Comment:
`$(brew --prefix)/opt/ruby/bin/gem` is probably better for this, since it would be automatic

Reply:
Sure thing - I'll update the other lines with this expression as well.